src/hooks/use-subscription-data.tsx

```diff
@@ -1,27 +1,26 @@
 import * as React from 'react'
 import axios from 'axios'
 
+type SubscriptionData = {
+  portalUrl?: string
+  billingScheme: 'tiered' | 'per_unit'
+  subscription?: any
+  price?: any
+  product?: any
+  latestInvoice?: any
+  upcomingInvoice?: any
+}
```

Comment:
I added some typing for the SubscriptionData to get some better type inference and warnings.